### ISSUE TITLE
PHP5.6 not working for osx 10.14.6 Mojava

### ISSUE THREAD

**Issue by gui0506** [eXolnet/homebrew-deprecated#22]
Hi there, 

I tried installing php@5.6 but seems like the package is broken. Or maybe I am missing some library there. Here is what I get:

```
/usr/local/Cellar/php@5.6/5.6.40/bin/php /Users/xxx/workspace/packagist-test/play.php
dyld: Library not loaded: /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib
  Referenced from: /usr/local/Cellar/php@5.6/5.6.40/bin/php
  Reason: image not found

Process finished with exit code 134 (interrupted by signal 6: SIGABRT)
```

**Comment by drbyte** [comment]
When upgrading openssl@1.1 the older symlinks to the 1.0 files appear to get removed.

I've found this temporarily fixes it, as long as the 1.0 files are still present:

```sh
ln -s /usr/local/Cellar/openssl/1.0.2t/lib/libcrypto.1.0.0.dylib /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib
ln -s /usr/local/Cellar/openssl/1.0.2t/lib/libssl.1.0.0.dylib /usr/local/opt/openssl/lib/libssl.1.0.0.dylib
```

**Comment by pgrenaud** [comment]
Hi!

Thank you for your interest in this repository. Unfortunately, this repository will now be archived with no further actions. We are sorry for the inconvenience.

**Why are we closing this repository?** This repository was only meant as a temporary measure, not a permanent one. Its only purpose was to ease the transition, considering that formulae from the homebrew-core tap are removed almost the day they became unsupported by the vendor. We needed a few more months to allow us to upgrade the code base of the various projects we have. But it was always with the intention of doing those upgrades, not by relying on a repository to keep old php versions artificially alive. We do not condone the use of deprecated software that could lead to serious security vulnerabilities.

**Why are we not redirecting to another repository?** Redirecting to another repository could be interpreted as an endorsement of said repository. If we were to do such a thing, we would not do it without vetting it first. And we do not wish to put the time and energy required in a vetting process of a third party repository. As the reason why a vetting process would be required, consider this. Before installing a software library on all our developer computers from an untrusted source, we would need to make sure that this software library is free from any malicious code (Trojan, ransomware, etc.), both in the repository itself and in the packaged binaries (the homebrew bottles, if any).

Thank you for your understanding.
